### PR TITLE
Fetch data from FIM table for display purposes

### PR DESCRIPTION
**Pull request by rpkamp**
- [x] Review https://github.com/SURFnet/learning-analytics/pull/74 first

### CODE REVIEW

File: src/AppBundle/Repository/FimRepository.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+     * @param Recipe $recipe
+     * @param Course $course
+     * @param Period $period
+     * @return array [
+     *  'Test 1' => [
+     *      0.9,  // student 1
+     *      0.85, // student 2
+     *      0.7,  // student 3
+     *      etc
+     *  ]
```
Comment (seggen-ibuildings) [inline src/AppBundle/Repository/FimRepository.php]: Missing: second closing `]` 😛

File: src/AppBundle/Repository/FimRepository.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            }
+
+            $result[$score['label']]['scores'][] = $score['score'] / $numberOfQuestionsPerTest[$score['label']];
+        }
+
+        return $result;
+    }
+
+    /**
+     * Used to normalise scores per test.
```
Comment (seggen-ibuildings) [inline src/AppBundle/Repository/FimRepository.php]: Please change to "Gets the number of questions per test. This is used to normalise the scores per test."

File: src/AppBundle/Repository/FimRepository.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Course;
+use AppBundle\Entity\Period;
+use AppBundle\Entity\Recipe;
+use Doctrine\ORM\EntityRepository;
+
+final class FimRepository extends EntityRepository
```
Comment (seggen-ibuildings) [inline src/AppBundle/Repository/FimRepository.php]: Just as a reminder: let's rename this class to `FimScoreRepository` before we add methods related to other aggregations.

File: app/config/services/repositories.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    app.repo.user:
+        class: Doctrine\ORM\EntityRepository
+        factory: ['@doctrine.orm.entity_manager', getRepository]
+        arguments: [AppBundle\Entity\User]
+        public: false
+
+    app.repo.queue:
+        class: AppBundle\Repository\TrackCommandRepository
+        factory: ['@doctrine.orm.entity_manager', getRepository]
+        arguments: [AppBundle\Entity\TrackCommand]
```
Comment (seggen-ibuildings) [inline app/config/services/repositories.yml]: 💯

File: src/AppBundle/Repository/FimRepository.php (hunk 1 above)
Comment (rpkamp) [inline src/AppBundle/Repository/FimRepository.php]: oh dear me! fixed :smile:

File: src/AppBundle/Repository/FimRepository.php (hunk 2 above)
Comment (rpkamp) [inline src/AppBundle/Repository/FimRepository.php]: Nice. done.

File: src/AppBundle/Repository/FimRepository.php (hunk 3 above)
Comment (rpkamp) [inline src/AppBundle/Repository/FimRepository.php]: Yup. Done.

**Pull request merged**